### PR TITLE
replacing npm Enterprise with npm On-Site

### PR DESCRIPTION
**Pull request by bcoe**
We should make sure that we land this immediately before landing the changes in copy for the website.

@cowperthwait let's hold off on any any copy edits to npmE emails, until we've landed this.

### CODE REVIEW

File: html-emails/application.html [hunk 1]
```diff
@@ -807,7 +807,7 @@
 		                  <tr>
 		                    <td class="wrapper last">
 													<h2>hello,<br>jonathan</h2>
```
Comment (cowperthwait) [inline html-emails/application.html]: “jonathan”?

File: html-emails/application.html [hunk 2]
```diff
@@ -807,7 +807,7 @@
...
-													<p>Thanks for trying out npm Enterprise, we're excited to have you aboard! To get started, please click below to verify your email address:</p>
+													<p>Thanks for trying out npm On-Site, we're excited to have you aboard! To get started, please click below to verify your email address:</p>
```
Comment (cowperthwait) [inline html-emails/application.html]: Suggested:
`Thanks for trying npm On-Site. We’re excited to have you aboard!`
(Would replace comma-splice with a full stop.)

**Comment by cowperthwait** [comment]
rest is fine

File: html-emails/application.html (hunk 1 above)
Comment (bcoe) [inline html-emails/application.html]: This was already the copy, was hoping you might know what's going on here? I didn't know we had HTML emails.

Comment (cowperthwait) [inline html-emails/application.html]: That makes two of us; but “[WIP]” suggests this is a work in progress by @ncawthon, who’s drafting up HTML templates for future transactional use in Eloqua. I hadn’t realized he was keeping his work here.

**Comment by ncawthon** [comment]
sorry if I put these the wrong place...

**Comment by aredridel** [comment]
:sheep: it!

**Pull request merged**